omas/omas_physics.py

```diff
@@ -1911,6 +1913,17 @@ def transform_current(rho, JtoR=None, JparB=None, equilibrium=None, includes_boo
 
     return Jout
 
+@add_to__ODS__
+def get_j_actuator_from_core_sources(ods, time_index=0):
```

Comment:
The function should also be renamed as `core_sources_get_j_actuator` (or better, `core_sources_j_parallel_sum`), and the docstring is missing.